order/Knaster_Tarski.lean

```diff
@@ -0,0 +1,100 @@
+import order.complete_lattice order.order_iso order.fixed_points
+
+open lattice
+
+universe u
+
+variables {α : Type u} [complete_lattice α]
+variables (f : α → α) (M : monotone f)
+
+def fixed_points : set α := { x | f x = x }
+
```

Comment:
I guess you want a lemma:
`lemma fixed_points.eq {x} (h : x \in fixed_points f M) : f x = x := h`
and use `.2.eq` this instead of `.2` in the further proofs, then you can use `simp` and `rw`..